src/@ionic-native/plugins/base64-to-gallery/index.ts

```diff
@@ -23,7 +23,7 @@ import { Cordova, Plugin, IonicNativePlugin } from '@ionic-native/core';
 @Plugin({
   pluginName: 'Base64ToGallery',
   plugin: 'cordova-base64-to-gallery',
-  pluginRef: 'cordova',
+  pluginRef: 'cordova.base64ToGallery',
```

Comment:
This should stay as `cordova` only. 

`cordova.base64ToGallery` is a function, not an object. Our wrapper wouldn't work unless if the pluginRef is set to `cordova`.